tools/test.py

```diff
@@ -471,6 +474,10 @@ def UnexpectedOutput(self):
       outcome = PASS
     return not outcome in self.test.outcomes
 
+  def HasSkipped(self):
```

Comment:
followed the conventions already established in test.py. not keen on rewriting them all?